lib/views/teach_word_view.dart

```diff
@@ -0,0 +1,356 @@
+import 'package:flutter/material.dart';
```

Comment:
Reorder the imports to alphabetical order

Reply:
please tell me it's not sorted correctly